governance/third-generation/aws/require-private-acl-and-kms-for-s3-buckets.sentinel

```diff
@@ -22,7 +22,7 @@ nonEncryptedS3Buckets = plan.filter_attribute_is_not_value(allS3Buckets,
 "aws:kms", true)
 
 # Main rule
-validated = nonPrivateS3Buckets["length"] is 0 and nonEncryptedS3Buckets["length"] is 0
+validated = length(nonPrivateS3Buckets["messages"]) is 0 and length(nonEncryptedS3Buckets["messages"]) is 0
 main = rule {
   validated is true
```

Comment:
suggest moving line 25 to within the main function, following the pattern in other rule examples

Reply:
@apolloclark : as we discussed via Slack, you meant the "main rule".  I don't like to evaluate compound boolean conditions in the main rule because that generates extra TRUE/FALSE lines from Sentinel's default output which I try to suppress as much as possible.